android/AndroidManifest.xml

```diff
@@ -103,8 +104,9 @@
         <category android:name="android.intent.category.DEFAULT"/>
         <category android:name="android.intent.category.BROWSABLE"/>
 
-        <data android:scheme="http" android:host="omaps.app"/>
+        <data android:scheme="http"/>
         <data android:scheme="https"/>
+        <data android:host="omaps.app"/>
```

Comment:
If several hosts can be combined, then we can merge omaps.app with ge0.me. Just leave a comment for ge0.me that it's a legacy from maps.me.

Reply:
OK, I will merge them together.

Reply:
Merged.